src/app/workbasket/filters/workbasket-filters.component.spec.ts

```diff
@@ -1,1058 +0,0 @@
-    beforeEach(async(() => {
-      workbasketHandler = createSpyObj('workbasketHandler', ['applyFilters']);
-      router = createSpyObj<Router>('router', ['navigate']);
-      router.navigate.and.returnValue(Promise.resolve('someResult'));
-      alertService = createSpyObj<AlertService>('alertService', ['isPreserveAlerts', 'setPreserveAlerts']);
-      orderService = createSpyObj('orderService', ['sortAsc']);
-      workbasketInputFilterService = createSpyObj<WorkbasketInputFilterService>('workbasketInputFilterService', ['getWorkbasketInputs']);
-      workbasketInputFilterService.getWorkbasketInputs.and.returnValue(createObservableFrom(TEST_WORKBASKET_INPUTS));
-      jurisdictionService = new JurisdictionService();
-      windowService = createSpyObj('windowService', ['setLocalStorage', 'getLocalStorage']);
-      windowService.getLocalStorage.and.returnValue('{}');
-      resetCaseTypes(JURISDICTION_2, CASE_TYPES_2);
-      activatedRoute = {
-        queryParams: Observable.of({}),
-        snapshot: {
-          queryParams: {}
-        }
-      };
-
-      TestBed
-        .configureTestingModule({
-          imports: [
-            FormsModule,
-            ReactiveFormsModule
-          ],
-          declarations: [
-            WorkbasketFiltersComponent,
-            FieldWriteComponent
-          ],
-          providers: [
-            { provide: Router, useValue: router },
-            { provide: ActivatedRoute, useValue: activatedRoute },
-            { provide: OrderService, useValue: orderService },
-            { provide: WorkbasketInputFilterService, useValue: workbasketInputFilterService },
-            { provide: JurisdictionService, useValue: jurisdictionService },
-            { provide: AlertService, useValue: alertService },
-            { provide: WindowService, useValue: windowService },
-          ]
-        })
-        .compileComponents()
-        .then(() => {
-          fixture = TestBed.createComponent(WorkbasketFiltersComponent);
-          component = fixture.componentInstance;
-
-          component.jurisdictions = [
-            JURISDICTION_1,
-            JURISDICTION_2
-          ];
-          component.formGroup = TEST_FORM_GROUP;
-          component.defaults = {
-            jurisdiction_id: JURISDICTION_2.id,
-            case_type_id: DEFAULT_CASE_TYPE.id,
-            state_id: DEFAULT_CASE_STATE.id
-          };
-          component.onApply.subscribe(workbasketHandler.applyFilters);
-
-          de = fixture.debugElement;
-          fixture.detectChanges();
-        });
-    }));
-
-    it('  ', async(() => {
-      component.selected.jurisdiction = JURISDICTION_2;
-      component.selected.caseType = null;
-      component.selected.caseState = null;
-
-      fixture.detectChanges();
-      let button = de.query($APPLY_BUTTON);
-      expect(button.nativeElement.disabled).toBeTruthy();
-    }));
-
-    it('should have an Apply button disabled when case type is not set', async(() => {
```

Comment:
Some other tests like this one assessing the buttons being disabled are missing in `case-list-filters.component.spec.ts`. Are we sure we do not want to keep them?